src/engine/sidechain/shoutconnection.cpp

```diff
@@ -837,10 +834,16 @@ void ShoutConnection::updateMetaData() {
             // setting both fields for MP3, tested players do not show anything different.
             // Also I do not know about icecast1. To be safe, i stick to the
             // old way for those use cases.
+            const QString artist = m_pMetaDataTrack->getArtist();
+            const QString title = m_pMetaDataTrack->getTitle();
             if (!m_format_is_mp3 && m_protocol_is_icecast2) {
-            	setFunctionCode(9);
-            	insertMetaData("artist", encodeString(artist).constData());
-            	insertMetaData("title", encodeString(title).constData());
+                kLogger.info() << "Updating metadata for"
+                               << profile()->getProfileName()
+                               << ": artist:" << m_customArtist
+                               << "| title:" << m_customTitle;
+                setFunctionCode(9);
```

Comment:
Any chance to move this `9` as a `kOnstant` or an `enum` which is a bit more explicative of what that value means?